runtimes/kotlin-1.6/src/main/kotlin/io/openruntimes/kotlin/Server.kt

```diff
@@ -28,6 +29,7 @@ suspend fun main() {
         .patch("/*") { runBlocking { execute(it) } }
         .options("/*") { runBlocking { execute(it) } }
         .head("/*") { runBlocking { execute(it) } }
+        .addHandler(HandlerType.TRACE, "/*") { runBlocking { execute(it) } }
```

Comment:
as per the docs, https://javalin.io/documentation#handlers you can use `trace()` directly.

Reply:
Docs mention supporting `TRACE`, however the routing api doesn't have a dedicated `trace()` method. I actually tried it earlier, the tests failed as there was no method. Ref - https://github.com/open-runtimes/open-runtimes/pull/242/commits/5f6085239ea45cbca691eca2fd6bf56cd80658d5